orchagent/orchdaemon.cpp

```diff
@@ -561,6 +561,17 @@ bool OrchDaemon::warmRestoreAndSyncUp()
         o->postBake();
```

Comment:
>postBake [](start = 11, length = 8)

This is special to MirrorOrch. Although it is not your code, I am wondering if possible to make it similar to other Orch, and we could add one more iteration to above for loop to cover both MirrorOrch and gAclOrch.

Reply:
I took a closer look at the code and ended up with (I think) what you and @wendani were suggesting. It does seem a lot cleaner now, but let me know if you had something else in mind!

Reply:
Offline discussed, we should explore the possibility to remove `postBake()` interface completely.
To postpone the MirrorOrch processing in warmboot, following below steps:

1. loop all other Orch(s) several rounds of `doTask()` in order to drain the consumer data
2. `MirrorOrch::doTask()`
3. `AclOrch doTask()`


---
In reply to: [519059988](https://github.com/Azure/sonic-swss/pull/1486#discussion_r519059988) [](ancestors = 519059988)